### ISSUE TITLE
Unexpected ddcutil output with nvidia card

### ISSUE THREAD

**Issue by mrpink17** [davidhi7/ddcci-plasmoid#32]
I own an nvidia GT440 and and ddcutil outputs this additional string each time it runs:
`(is_nvidia_einval_bug          ) nvida/i2c-dev bug encountered. Forcing future io I2C_IO_STRATEGY_FILEIO. Retrying` 

The code which parses the output of ddcutil fails when it encounters the nvidia warning.
To make it work on my machine I patched `async_subprocess_wrapper` and `subprocess_wrapper` functions:
`stdout = stdout.decode().replace('(is_nvidia_einval_bug          ) nvida/i2c-dev bug encountered. Forcing future io I2C_IO_STRATEGY_FILEIO. Retrying\n', '')`

**Comment by mrpink17** [comment]
This is the output I get with the unpatched version of ddcci.py:
```
[mrpink@i5k ~]$ python -m ddcci_plasmoid_backend --debug detect
DEBUG __main__: Running version 0.1.5 in debug mode
DEBUG __main__: Command: detect
DEBUG ddcci_plasmoid_backend.ddcci: Execute command: `ddcutil detect`
DEBUG ddcci_plasmoid_backend.ddcci: [code]   ddcutil detect: 0
DEBUG ddcci_plasmoid_backend.ddcci: [stdout] ddcutil detect: (is_nvidia_einval_bug          ) nvida/i2c-dev bug encountered. Forcing future io I2C_IO_STRATEGY_FILEIO. Retrying
Display 1
   I2C bus:  /dev/i2c-2
   DRM connector:           card0-VGA-1
   EDID synopsis:
      Mfg id:               SAM - Samsung Electric Company
      Model:                SMBX2035
      Product code:         1789  (0x06fd)
      Serial number:        H9MB200687
      Binary serial number: 808464697 (0x30303139)
      Manufacture year:     2011,  Week: 7
   VCP version:         2.0
DEBUG ddcci_plasmoid_backend.ddcci: [stderr] ddcutil detect: 

DEBUG ddcci_plasmoid_backend.ddcci: Found 2 entries at root level
DEBUG ddcci_plasmoid_backend.ddcci: Key (is_nvidia_einval_bug          ) nvida/i2c-dev bug encountered. Forcing future io I2C_IO_STRATEGY_FILEIO. Retrying does not match pattern for valid display, so skip it
DEBUG ddcci_plasmoid_backend.ddcci: Execute command: `ddcutil getvcp --bus 2 --brief 10`
DEBUG ddcci_plasmoid_backend.ddcci: [code]   ddcutil getvcp --bus 2 --brief 10: 1
DEBUG ddcci_plasmoid_backend.ddcci: [stdout] ddcutil getvcp --bus 2 --brief 10: (is_nvidia_einval_bug          ) nvida/i2c-dev bug encountered. Forcing future io I2C_IO_STRATEGY_FILEIO. Retrying
VCP 10 C 14 100
DEBUG ddcci_plasmoid_backend.ddcci: [stderr] ddcutil getvcp --bus 2 --brief 10: 

DEBUG __main__: too many values to unpack (expected 5)
DEBUG __main__: Detected 0 working monitor busses, 1 non-working bus.
{"command": "detect", "value": []}
```

**Comment by davidhi7** [comment]
Fixed in release 0.1.6, I hope you don't mind that I simply copied the code you published here. Thanks for the report!
